internal_ws/ykbh/src/lib.rs

```diff
@@ -281,8 +291,8 @@ impl SIRInterpreter {
                 self.frames.push(frame);
             }
             Terminator::Return => {
-                // Return from current stackframe.
                 let oldframe = self.frames.pop().unwrap();
+                // XXX comment could do with rephrasing
```

Comment:
Something like "If there are no further frames, then we are back at the control point".

Reply:
Fixed in df14c2a.